components/card/card_header.tsx

```diff
@@ -7,13 +7,44 @@ import classNames from 'classnames';
 type Props = {
     children: React.ReactNode;
     expanded?: boolean;
+    title?: React.ReactNode;
+    subtitle?: React.ReactNode;
+    buttonText?: React.ReactNode;
+    onClick?: () => void;
 };
 
 const CardHeader: React.FC<Props> = (props: Props) => {
     return (
         <div className={classNames('Card__header', {expanded: props.expanded})}>
-            {props.children}
-            {props.expanded && <hr className='Card__hr'/>}
+            {
+                props.title && props.subtitle ?
```

Comment:
This feels a bit weird to me, architecturally. The aim of the `CardHeader` component is to not be prescriptive about what goes in the header, letting the consumer dictate the direction of what's inside. The way your changes are setup looks to short circuit that by giving the user an alternate method of filling the header that the component provides.

I'm thinking here that we should simply provide either an alternate version of `CardHeader` that only takes the title and subtitle as parameters and generates the HTML you've added, OR we create a component that you can pass into `CardHeader` as a child that does the same.